aries-site/src/examples/templates/filtering/FilteringCards/FilteringCards.js

```diff
@@ -63,36 +63,40 @@ export const FilteringCards = ({ containerRef }) => {
       <Heading level={2} margin="none">
         Users
       </Heading>
-      <FiltersProvider>
-        <Box gap="medium">
-          <FilterControls
-            data={users}
-            filters={filtersConfig}
-            layerProps={layerProps}
-            searchFilter={{ placeholder: 'Search' }}
-          />
+      <Box fill>
+        <Data
+          data={users}
+          updateOn="submit"
+          height={{ min: 'medium', max: '100%' }}
+        >
+          <Toolbar>
+            <DataSearch />
+            <DataFilters layer={layerProps}>
+              <DataFilter property="role" label="Role" />
+              <DataFilter property="status" label="Status" />
+              <DataFilter property="location" label="Location" />
+              <DataFilter property="hoursAvailable" label="Hours Available" />
+              <DataFilter property="name" label="Name" />
+            </DataFilters>
+          </Toolbar>
+          <DataSummary />
           <Users />
-        </Box>
-      </FiltersProvider>
+        </Data>
+      </Box>
     </Box>
   );
 };
 
 const Users = () => {
   const size = useContext(ResponsiveContext);
-  const { filteredResults } = useFilters();
 
   return (
-    <Box overflow="auto" fill>
-      <Grid
-        columns={
-          !['xsmall', 'small'].includes(size)
-            ? 'small'
-            : { count: 2, size: 'auto' }
-        }
+    <Box flex overflow="auto">
```

Comment:
Why do we change from `fill` to `flex`?

Reply:
Didn't catch that, changed back to `fill`

Reply:
I am still seeing `flex`.